LDAR_Sim/testing/unit_testing/test_initialization/test_leaks/leak_testing_fixtures.py

```diff
@@ -0,0 +1,83 @@
+""" Fixtures for testing leaks"""
+
+import pytest
+from datetime import datetime
+from typing import Any, Dict
+
+
+@pytest.fixture(name="mock_program")
+def mock_program_fix():
+    return {
+        'subtype_file': None,
+        'NRd': 10,
+        'n_init_leaks': None,
+        'n_init_days': None,
+        'start_date': (2022, 1, 1),
+        'emissions': {
+            'LPR': 0.5,
+            'leak_file': None,
+        },
+        'leak_rate_source': 'sample',
+        'empirical_leak_rates': [1],
+
+    }
+
+
+@pytest.fixture(name="mock_program_2")
+def mock_program_2_fix():
+    return {
+        'subtype_file': None,
+        'NRd': 10,
+        'n_init_leaks': 1,
+        'n_init_days': 10,
+        'start_date': (2022, 1, 1),
+        'emissions': {
+            'LPR': 0.5,
+            'leak_file': None,
+        },
+        'leak_rate_source': 'sample',
+        'empirical_leak_rates': [1],
+    }
+
+
+@pytest.fixture(name="mock_site_for_leak_test")
+def mock_site_for_leak_test_fix() -> Dict[str, Any]:
+    return {
+        'facility_ID': 'test',
+        'equipment_groups': 1,
+        'empirical_leak_rates': [1],
+        'lat': 52.0,
+        'lon': -114.0,
+        'leak_rate_source': 'sample',
+    }
+
+
+@pytest.fixture(name="mock_site_w_leak_test")
```

Comment:
[Nit] Is this the fixture for the expected result? If so can we put expected or something in the name to make that clear?

Reply:
done.